bundles/org.openhab.binding.openuv/src/main/java/org/openhab/binding/openuv/internal/handler/OpenUVBridgeHandler.java

```diff
@@ -93,11 +96,8 @@ public void initialize() {
 
     @Override
     public void dispose() {
-        ScheduledFuture<?> job = this.reconnectJob;
-        if (job != null && !job.isCancelled()) {
-            job.cancel(true);
-        }
-        reconnectJob = null;
+        reconnectJob.ifPresent(job -> job.cancel(false));
```

Comment:
You switch from true to false. Why ?

Reply:
Because this can cause the same InterruptedException we had on Netatmo binding.

Reply:
As cancel(true) is used almost everywhere, I need to think why false us required here. 
And I did not take about this change in the netatmo binding. I should have. 
I do not say this change is wrong, I just want to be sure and understand.

Reply:
In Netatmo binding, true led to InterruptedException in the http request procedure, leading to unpredictable situation.
It is not a common issue in openuv binding because it is far more simple, but while debugging in searching the origin of this bug, I saw it. But I can revert if you prefer.

Reply:
I see that you revert it to cancel(true). So I merge without delay the PR but I think we should clearly have a conviction if true or false should be used.